### PR TITLE
removing event listener for now

### PR DESCRIPTION
**Pull request by brycedorn**
looks like this was autoplaying first video regardless of the second video that is selected. can revisit afterward for followup fix, this (should) just disable annoying audio.

corresponding omni pr: https://github.com/theonion/omni/pull/899

### CODE REVIEW

File: elements/bulbs-video/elements/video-carousel.js [hunk 1]
```diff
@@ -57,7 +57,10 @@ class BulbsVideoCarousel extends BulbsHTMLElement {
       '<bulbs-video-carousel> MUST contain a <bulbs-carousel>'
     );
 
-    this.videoPlayer.addEventListener('jw-beforePlay', this.firstPlay = this.firstPlay.bind(this), true);
+    // Removing for now, causing issue with videos autoplaying in background 
+    //  - bryce, 2/22/17
+    // this.videoPlayer.addEventListener('jw-beforePlay', this.firstPlay = this.firstPlay.bind(this), true);
```
Comment (spra85) [inline elements/bulbs-video/elements/video-carousel.js]: @collin can you weigh in here?

**Comment by brycedorn** [comment]
FWIW, is a bug on ON homepage as well.

File: elements/bulbs-video/elements/video-carousel.js (hunk 1 above)
Comment (collin) [inline elements/bulbs-video/elements/video-carousel.js]: I think this is listening to the first play event to update the styling on the carousel items.

File: elements/bulbs-video/elements/video-carousel.js [hunk 2]
```diff
@@ -68,8 +71,9 @@ class BulbsVideoCarousel extends BulbsHTMLElement {
 
   firstPlay () {
     let items = this.querySelectorAll('bulbs-carousel-item');
+    let nowPlayingItems = this.querySelectorAll('[now-playing]');
 
-    if (items.length > 0) {
+    if (nowPlayingItems.length > 0 && items.length > 0) {
```
Comment (collin) [inline elements/bulbs-video/elements/video-carousel.js]: Which calls through to this. The apply state function is doing two things:

1: updating the styles on the carousel item to highlight the current playing item.
2: sets `src` attribute of the video player. Changing the `src` attribute of a `<bulbs-video>` tears down the jwplayer instance and creates a new one.

File: elements/bulbs-video/elements/video-carousel.js (hunk 1 above)
Comment (collin) [inline elements/bulbs-video/elements/video-carousel.js]: Losing that styling for a moment is probably better than playing two stacked videos as a bug.

File: elements/bulbs-video/elements/video-carousel.js (hunk 2 above)
Comment (brycedorn) [inline elements/bulbs-video/elements/video-carousel.js]: but the `selectItem` call below is currently forcing `items[0]` to play, regardless of which one is selected. the highlight styles don't seem to be related to `firstPlay`, as this is only called once (and then the eventlistener is removed), so I'm not really sure what the intended functionality of this is

**Comment by brycedorn** [comment]
merging with plans to followup with better soln soon.
